crates/autopilot/src/driver_model.rs

```diff
@@ -155,15 +155,17 @@ pub mod execute {
     #[derive(Clone, Debug, Default, Deserialize)]
     #[serde(rename_all = "camelCase", deny_unknown_fields)]
     pub struct Response {
-        pub account: H160,
-        pub nonce: u64,
-        #[serde_as(as = "BTreeMap<_, DisplayFromStr>")]
-        pub clearing_prices: BTreeMap<H160, U256>,
-        pub trades: Vec<Trade>,
-        pub internalized_interactions: Vec<InternalizedInteraction>,
```

Comment:
I think we still need to keep these around. I think the information of the expected input and output amounts gets lost with just the `Calldata` return type as is.

Reply:
These were never actually returned from the driver. Can you specify exactly what the response of `/settle` should be?

Reply:
Also note that, the way it's written now, the `/settle` endpoint will only return after the settlement has been confirmed. I believe this is actually wrong, since the autopilot waits for settlement confirmation? This needs to be well-defined.

Reply:
Let me double-check.

Reply:
>  I believe this is actually wrong,

I think so too, I think only the `autopilot` needs to wait for it to mine for returning. That being said, the `driver` should still spawn a background task for the mining (it wants to potentially cancel, resubmit with higher gas price, etc.)

Reply:
> Can you specify exactly what the response of /settle should be?

Sorry, the input and output amounts actually don't matter. The format you defined is correct. My bad.

Reply:
What if there is custom driver implementation where the `settle` response is sent back to autopilot just after the settlement is mined? In that case, receiving `call_data` as part of `settle` response is too late for autopilot, since autopilot must save `call_data` to DB before settlement is mined.